.github/workflows/ci.yml

```
@@ -100,3 +101,56 @@ jobs:
         with:
           publish-delay: 30000
           registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+
+  publish-docker-image:
+    needs:
+      - cargo-verifications
+    runs-on: ubuntu-latest
```

we can speed up this job by using buildjet runners

```suggestion
    runs-on: buildjet-4vcpu-ubuntu-2204
```

---

@Voxelot should we do this for all the actions?

---

sure